docs/developer_guide/debugging.rst

```diff
@@ -0,0 +1,73 @@
+Debugging
+=========
+
+This pages contains some Mantid Imaging specific debugging tips.
+
+Getting debug output
+--------------------
+
+There are several options that can be used so that Mantid Imaging outputs more information as it runs, for example::
+
+  python -X faulthandler -W default -m mantidimaging --log-level DEBUG
+
+This enables the :py:mod:`faulthandler` to dump a traceback in the event of python crash. Causes python :py:mod:`warnings` to be printed. And sets the Manitd Imaging :py:mod:`log level <logging>` to DEBUG.
```

Comment:
Typo: And sets the Mantid Imaging

Reply:
I've now changed it to bullet points to avoid it being a long sentence.